src/core/core.controller.js

```diff
@@ -985,8 +1014,12 @@ helpers.extend(Chart.prototype, /** @lends Chart */ {
 		// Find Active Elements for hover and tooltips
 		if (e.type === 'mouseout') {
 			me.active = [];
+			me._lastEvent = null;
 		} else {
 			me.active = me.getElementsAtEventForMode(e, hoverOptions.mode, hoverOptions);
+			if (e.type !== 'click') {
```

Comment:
When the [pen](https://codepen.io/Evertvdw/pen/PgqbWq) is paused, clicking legend to show/hide dataset has no effect (or in pie until you hover the chart). I think changing this to `me._lastEvent = e.type === 'click' ? null : e;` might fix that.

Reply:
Yes it does fix that. I updated the code and the codepen.